src/components/Dropdown/Dropdown.scss

```diff
@@ -18,8 +20,14 @@
       height: 24px;
       flex-grow: 1;
       display: flex;
-      justify-content: flex-end;
+      justify-content: space-between;
```

Comment:
now that the drop down icon is an actual DOM element, needed to apply spacing between selected option and the icon else it looks ugly and bunched up.